lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -1941,7 +1941,72 @@ my %stats = (
                 PREREQ_ONLY => 1
             }
         } MusicBrainz::Server::Data::Relationship->all_pairs
-    )
+    ),
+
+    "count.collection" => {
+        DESC => "Count of all collections",
+        SQL => "SELECT COUNT(*) FROM editor_collection",
+    },
+
+    "count.collection.type.release" => {
+        CALC => sub {
+            my ($self, $sql) = @_;
+
+            my $data = $sql->select_list_of_lists(
+                "SELECT type, COUNT(*) AS count
+                FROM editor_collection
+                GROUP BY type
+                ",
```

Comment:
The hanging `",` looked a little funny so I thought I'd suggest using `<<~'SQL'` here if you feel like it.

Reply:
Made that change - a lot of these could use SQL blocks tbh, but dunno if you want me doing it now? I could add it as a second commit.

Reply:
Maybe not all in this PR, but there are some more being added here that could be changed.